Ginger/Ginger/ApplicationModelsLib/ModelOptionalValue/ImportOptionalValuesForParameters.cs

```diff
@@ -935,51 +935,10 @@ private string ExportParametersToFile(List<AppParameters> parameters, string fil
         {
             try
             {
-                int colCount = 100;
-                foreach (var paramVal in parameters)
-                {
-                    if (paramVal.OptionalValuesString.Contains(CURRENT_VAL_PARAMETER))
-                    {
-                        colCount = (paramVal.OptionalValuesList.Count - 1) > colCount ? (paramVal.OptionalValuesList.Count - 1) : colCount;
-                    }
-                    else
-                    {
-                        colCount = (paramVal.OptionalValuesList.Count) > colCount ? (paramVal.OptionalValuesList.Count) : colCount;
-                    }
-                }
-
                 string tableName = fileName.StartsWith("GlobalParameters") ? "GlobalModelParameters" : "ModelParameters";
-                DataTable dtTemplate = new DataTable(tableName);
-                dtTemplate.Columns.Add(PARAMETER_NAME, typeof(string));
-                dtTemplate.Columns.Add(DESCRIPTION, typeof(string));
 
```

Comment:
pretty breakable to go with this filename check, better to use some context enum